config.json.sample

```diff
@@ -4,25 +4,60 @@
     "password": "12345"
   },
   "locations": [
-    "Valeyre",
     "Suzanne Lenglen",
-    "Poliveau"
-  ],
-  "date": "9/10/2021",
-  "hours": [
-    "14",
-    "15",
-    "16",
-    "10",
-    "11"
-  ],
-  "priceType": [
-    "Tarif plein",
-    "Tarif réduit"
+    "Niox",
+    "Henry de Montherlant",
+    "Valeyre",
+    "René et André Mourlon",
+    "Rigoulot - La Plaine",
+    "Paul Barruel",
+    "Sablonnière",
+    "Croix Nivert",
+    "Atlantique",
+    "Elisabeth",
+    "Cordelières",
+    "NEUVE SAINT PIERRE",
+    "Poliveau",
+    "Candie",
+    "Philippe Auguste",
+    "Thiéré",
+    "Alain Mimoun",
+    "Carnot",
+    "La Faluère",
+    "Léo Lagrange",
+    "Château des Rentiers",
+    "Dunois",
+    "Georges Carpentier",
+    "Moureu - Baudricourt",
+    "Poterne des Peupliers",
+    "Puteaux",
+    "Aurelle de Paladines",
+    "Broquedis - Asnières",
+    "Courcelles",
+    "Max Rousié",
+    "Bertrand Dauvin",
+    "Halle Fret",
+    "Jesse Owens",
+    "Poissonniers",
+    "Bobigny",
+    "Edouard Pailleron",
+    "Jandelle",
+    "Jules Ladoumègue",
+    "Sept arpents",
+    "Amandiers",
+    "Docteurs Déjerine"
```

Comment:
Good job for listing all court, do you think we can order them alphabetically for easier selection for user?

Reply:
Done ✅